src/applications/gi/reducers/calculator.js

```diff
@@ -51,10 +51,23 @@ export default function (state = INITIAL_STATE, action) {
         convertedValue = +value.replace(/[^0-9.]+/g, '');
       }
 
-      const newState = {
+      let newState = {
         [field]: convertedValue,
       };
 
+      if (field === 'yellowRibbonDegreeLevel') {
+        const yellowRibbonDivisionOptions = state.yellowRibbonPrograms.length > 0 ?
+          [...new Set(state.yellowRibbonPrograms
```

Comment:
Not sure what the preferred way to create a unique value array is- I could have pulled in a lodash function to create a unique array. `Set` is optimized for iteration and doesn't have an easy way to pull the first value.